src/keboola/component/dao.py

```diff
@@ -849,6 +849,15 @@ def __init__(self, name: str,
         self.delete_where_column = None
         self.delete_where_operator = None
 
+        if kwargs.get('delete_where_values'):
```

Comment:
@kudj Jeste jedna vec, nez tady ocekavat tyhle veci, tak bych radsi postavil ten filter manifest jako v tomhle testu https://github.com/keboola/python-component/blob/f264dcd78d5589495c4ee951b49b945005cdfbd2/tests/test_dao.py#L139 v metode `build_from_manifest`

Reply:
ok tak to nechejme ted. Je fakt, ze stejne nikdo asi nebude volat TableDefinition a cpat tam jak ten dict tak nejakej ten parametr.